InformationScripting/src/graph/PropertyMap.cpp

```diff
@@ -41,4 +47,19 @@ Property InformationScripting::PropertyMap::operator[](const QString& key) const
 	Q_ASSERT(false);
 }
 
+Property& InformationScripting::PropertyMap::operator[](const QString& key)
+{
+	for (auto& content : properties_)
+		if (content.first == key) return content.second;
+	qDebug() << "No object with name" << key;
```

Comment:
Well, I'm not sure I agree. Why would it not be suitable now, but suitable later? We should decide what to do and do it from now so that we get to test the code the way it's meant to be. Unless there is a compelling reason to leave it as it is, I would rather change it to the more standard behavior.